### PR TITLE
[Feature] use "set warehouse xxx" instead of "use 'warehouse xxx'" when specify a warehouse

### PR DESCRIPTION
**Pull request by abc982627271**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java [hunk 1]
```diff
@@ -288,6 +288,9 @@ private void addRunningQueryDetail(StatementBase parsedStmt) {
 
     // process COM_QUERY statement,
     private void handleQuery() {
+        // for debug
+        LOG.info("enter handleQuery");
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java:292]: should remove this log

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 19 / 25 (76.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/WarehouseAnalyzer.java|4|10|40.00%|[43, 53, 63, 72, 80, 89]|
|:large_blue_circle:|com/starrocks/sql/ast/SetWarehouseStmt.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/Analyzer.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/SuspendWarehouseStmt.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|5|5|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java [hunk 2]
```diff
@@ -471,10 +471,10 @@ public void execute() throws Exception {
                 }
             } else if (parsedStmt instanceof SetStmt) {
                 handleSetStmt();
-            } else if (parsedStmt instanceof UseWarehouseStmt) {
-                handleUseWarehouseStmt();
             } else if (parsedStmt instanceof UseDbStmt) {
                 handleUseDbStmt();
+            } else if (parsedStmt instanceof  SetWarehouseStmt) {
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java:476]:
```suggestion
            } else if (parsedStmt instanceof SetWarehouseStmt) {
```

File: fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java [hunk 3]
```diff
@@ -999,17 +999,20 @@ private void handleUseCatalogStmt() throws AnalysisException {
     }
 
     // Process use warehouse statement
-    private void handleUseWarehouseStmt() throws AnalysisException {
-        UseWarehouseStmt useWarehouseStmt = (UseWarehouseStmt) parsedStmt;
+    private void handleSetWarehouseStmt() throws AnalysisException {
+        // for debug
+        System.out.println("enter handleSetWarehouseStmt");
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java:1004]: remove

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/SuspendWarehouseStmt.java [hunk 4]
```diff
@@ -15,14 +15,14 @@
...
 public class SuspendWarehouseStmt extends DdlStmt {
-    private String whName;
+    private String warehouseName;
 
     public SuspendWarehouseStmt(String whName) {
-        this.whName = whName;
+        this.warehouseName = whName;
     }
 
     public String getFullWhName() {
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/sql/ast/SuspendWarehouseStmt.java:24]: why use full?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 15 / 21 (71.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/WarehouseAnalyzer.java|4|10|40.00%|[43, 53, 63, 72, 80, 89]|
|:large_blue_circle:|com/starrocks/sql/ast/SetWarehouseStmt.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/Analyzer.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|4|4|100.00%|[]|

**Review by wyb — DISMISSED** [review]

**Review by nshangyiming — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by gengjun-git — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 16 / 22 (72.73%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/WarehouseAnalyzer.java|4|10|40.00%|[43, 53, 63, 72, 80, 89]|
|:large_blue_circle:|com/starrocks/sql/ast/SetWarehouseStmt.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/Analyzer.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|4|4|100.00%|[]|

**Comment by abc982627271** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 16 / 22 (72.73%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/WarehouseAnalyzer.java|4|10|40.00%|[43, 53, 63, 72, 80, 89]|
|:large_blue_circle:|com/starrocks/sql/ast/SetWarehouseStmt.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/Analyzer.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|4|4|100.00%|[]|

**Review by wyb — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**